.changeset/sixty-pumpkins-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/linear-finance-adapter': major
+---
+
+update ea to use api instead of reading from csv file
```

Comment:
Just thought about this as well. Could we explicitly call out what has changed, making it easier to do the release notes for this?

E.g.:
```suggestion
Update ea to use api instead of reading from csv file. NOTE: This version of the adapter requires an API key to be set.
```